In [90]:
# headless 옵션 추가하기 ->

from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 


service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("headless")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
options.add_argument("--user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
driver = webdriver.Chrome(service = service, options = options) 
#------------------기본설정------------------------------

driver.get("https://davelee-fun.github.io/") 

element = driver.find_element(By.TAG_NAME,"h1")
print(element.text)

time.sleep(3)

driver.quit()

Teddy의 선물 블로그


#### 다음 웹사이트에 접속해서 셀레니움을 활용해서 크롤링해오세요.
- https://crawlingtest.netlify.app/
- 부모요소의 id 속성값이 dv_course_list인 자식요소의 class가 course인 데이터만 출력해주세요
- 

In [37]:
#문제 풀이 
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 


service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("headless")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
#options.add_argument("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
#사람처럼 보이게하는 방법 BUT Headless와 충돌하기 때문에 꺼줌
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://crawlingtest.netlify.app/") 


elements = driver.find_elements(By.CSS_SELECTOR,"#dev_course_list.course")
for element in elements:
    print(element.get_text())
time.sleep(3)
driver.quit()

In [42]:
#문제 선생님 해설 
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 


service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("headless")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
#options.add_argument("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
#사람처럼 보이게하는 방법 BUT Headless와 충돌하기 때문에 꺼줌
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://crawlingtest.netlify.app/") 


elements = driver.find_elements(By.CSS_SELECTOR,"ul#dev_course_list > li.course") # ul#으로 해야함!!
for element in elements:
    print(element.text)
time.sleep(3)
driver.quit()

(초급) - 강사가 실제 사용하는 자동 프로그램 소개 [2]
(초급) - 필요한 프로그램 설치 시연 [5]
(초급) - 데이터를 엑셀 파일로 만들기 [9]
(초급) -     엑셀 파일 이쁘게! 이쁘게! [8]
(초급) -     나대신 주기적으로 파이썬 프로그램 실행하기 [7]
(초급) - 파이썬으로 슬랙(slack) 메신저에 글쓰기 [40]
(초급) - 웹사이트 변경사항 주기적으로 체크해서, 메신저로 알람주기 [12]
(초급) - 네이버 API 사용해서, 블로그에 글쓰기 [42]
(중급) - 자동으로 쿠팡파트너스 API 로 가져온 상품 정보, 네이버 블로그/트위터에 홍보하기 [412]


#### 문제 2, 해당사이트에 접속한 뒤 커리큘럼 일점표 안의 데이터를 모두 추가하세

In [44]:
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 


service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("headless")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
#options.add_argument("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
#사람처럼 보이게하는 방법 BUT Headless와 충돌하기 때문에 꺼줌
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://crawlingtest.netlify.app/") 

elements = driver.find_elements(By.TAG_NAME,"tbody")
for element in elements:
    print(element.text)

일정 커리큘럼 타이틀 난이도
5.1 ~ 6.15 나만의 엣지있는 블로그 사이트 만들기 (취미로 익히는 IT) 초급
6.16 ~ 7.31 파이썬과 데이터과학 첫걸음 (IT 기본기 익히기) 중급


In [46]:
# 선생님 해설!!!!!!!!!!!!!
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("headless")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
#options.add_argument("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
#사람처럼 보이게하는 방법 BUT Headless와 충돌하기 때문에 꺼줌
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://crawlingtest.netlify.app/") 

# table 태그에는 thead, tbody, tfooter로 구성됨. 컨텐츠는 대부분 tbody에 있음
# tbody는 tr, td로 나뉨!

# elements = driver.find_elements(By.TAG_NAME,"td")
#선택자로 찾아와보기
elements = driver.find_elements(By.CSS_SELECTOR,"table tr td")
for element in elements:
    print(element.text)

일정
커리큘럼 타이틀
난이도
5.1 ~ 6.15
나만의 엣지있는 블로그 사이트 만들기 (취미로 익히는 IT)
초급
6.16 ~ 7.31
파이썬과 데이터과학 첫걸음 (IT 기본기 익히기)
중급


#### 동일한 웹사이트에서 아래 데이터만 추출해오세요.
중급) - 자동으로 쿠팡파트너스 API 로 가져온 상품 정보, 네이버 블로그/트위터에 홍보하기 [412]

In [69]:
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("headless")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
#options.add_argument("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
#사람처럼 보이게하는 방법 BUT Headless와 충돌하기 때문에 꺼줌
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://crawlingtest.netlify.app/") 

elements = driver.find_elements(By.TAG_NAME,"a")
print("text :" , elements[-1].text)

time.sleep(3)
driver.quit()

text : (중급) - 자동으로 쿠팡파트너스 API 로 가져온 상품 정보, 네이버 블로그/트위터에 홍보하기 [412]


#### 다음 웹 사이트에 접속해서 selenium을 활용해서 크롤링 해오세요
- https://divjason.github.io/sellenium-test/
- class 값이 news 인 요소의 태그 내용을 찾아서 출력해주세요!

In [62]:
#선생님 해설

from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("headless")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
#options.add_argument("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
#사람처럼 보이게하는 방법 BUT Headless와 충돌하기 때문에 꺼줌
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://crawlingtest.netlify.app/") 

elements = driver.find_elements(By.CSS_SELECTOR,"ul#dev_course_list > li.course")
print(elements[-1].text)

time.sleep(3)
driver.quit()

(중급) - 자동으로 쿠팡파트너스 API 로 가져온 상품 정보, 네이버 블로그/트위터에 홍보하기 [412]


In [72]:
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("headless")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
#options.add_argument("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
#사람처럼 보이게하는 방법 BUT Headless와 충돌하기 때문에 꺼줌
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://divjason.github.io/sellenium-test/") 

element = driver.find_element(By.CLASS_NAME,"news")
print(element.text)

time.sleep(3)
driver.quit()

뉴스: 머신러닝 강좌가 나왔어요~~~


In [73]:
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("headless")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
#options.add_argument("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
#사람처럼 보이게하는 방법 BUT Headless와 충돌하기 때문에 꺼줌
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://divjason.github.io/sellenium-test/") 

element = driver.find_element(By.TAG_NAME,"span")
print(element.text)

time.sleep(3)
driver.quit()

뉴스: 머신러닝 강좌가 나왔어요~~~


In [75]:
#선생님 해설

from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("headless")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
#options.add_argument("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
#사람처럼 보이게하는 방법 BUT Headless와 충돌하기 때문에 꺼줌
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://divjason.github.io/sellenium-test/") 

element = driver.find_element(By.CSS_SELECTOR,"span.news")
element = driver.find_element(By.TAG_NAME,"span")
print(element.text)

time.sleep(3)
driver.quit()

뉴스: 머신러닝 강좌가 나왔어요~~~


#### 다음 웹 사이트에 접속해서 selenium을 활용해서 크롤링 해오세요
- https://divjason.github.io/sellenium-test/
- input tag요소에 id 값으로는 error@error.com / password 값으로는 "비밀번호1234"를 입력 후 로그인 버튼 자동클릭하기

In [89]:
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys # 1, keys라는 인스턴트 객체를 찾아옴!
from webdriver_manager.chrome import ChromeDriverManager 
import time 
from selenium.webdriver.chrome.options import Options


service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
driver = webdriver.Chrome(service = service, options = options) 

driver.get("https://divjason.github.io/sellenium-test/") 

username = driver.find_element(By.NAME,"username")
password = driver.find_element(By.NAME,"password")
username.clear()
password.clear()

username.send_keys("error@error.com")
password.send_keys("비밀번호1234")

time.sleep(3)
password.send_keys(Keys.RETURN) #return은 enter랑 똑같은 의미!! 결과물제출.
time.sleep(3)

driver.quit()

In [102]:
# 선생님해설!
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
options.add_argument("--user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://divjason.github.io/sellenium-test/") 

element = driver.find_element(By.ID,"username")
element.clear()
element.send_keys("error@error.com")

element = driver.find_element(By.ID,"password")
element.clear()
element.send_keys("비밀번호1234")

#element요소를 두번써도 괜찮음!
#엔터를 누르면 바로 서버로보내라는 것이 내장. But 로그인을 할때 엔터를 치지못하게 만드는 이유?
#->prevent default 함수를 쓰면 리턴(엔터)키를 눌러도  접속되지않음. 이럴때는 명확하게 버튼을 찾아와서 버튼을 클릭하게 만들어야함!
# element.send_keys(Keys.RETURN)
#해결방법 =
element = driver.find_element(By.CSS_SELECTOR,"input[type='submit']") 
# CSS선택자에서 속성선택자를 찾아오는 형식 ->"input[type='submit']"
element.click() 
#정석!!


time.sleep(5)

driver.quit()

#### 다음 웹 사이트에 접속해서 selenium을 활용해서 크롤링 해오세요
- https://divjason.github.io/sellenium-test/
- input tag요소에 id 값으로는 error@error.com / password 값으로는 "비밀번호1234"를 입력 후 로그인 버튼 자동클릭하기
- 클릭 이후 이동한 페이지에서 message라는 클래스 속성값을 가지고 있는 요소의 텍스트 정보를 출력!

In [105]:
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
options.add_argument("--user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://divjason.github.io/sellenium-test/") 

element = driver.find_element(By.ID,"username")
element.clear()
element.send_keys("error@error.com")

element = driver.find_element(By.ID,"password")
element.clear()
element.send_keys("비밀번호1234")

element = driver.find_element(By.CSS_SELECTOR,"input[type='submit']") 
element.click() 
element1 = driver.find_element(By.CSS_SELECTOR,"div.message")
print(element1.text)

time.sleep(3)

driver.quit()

실제 사이트는 https://megaitlab.com/ 입니다
본 페이지는 로그인된 페이지입니다!
뉴스: 머신러닝 강좌가 나왔어요~~~


In [106]:
#선생님 해설! 로그인 + 정보값찾아오기!

from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
options.add_argument("--user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://divjason.github.io/sellenium-test/") 

element = driver.find_element(By.ID,"username")
element.clear()
element.send_keys("error@error.com")

element = driver.find_element(By.ID,"password")
element.clear()
element.send_keys("비밀번호1234")

element = driver.find_element(By.CSS_SELECTOR,"input[type='submit']") 
element.click() 
element1 = driver.find_element(By.CSS_SELECTOR,"div.message") #"태그.속성값"
print(element1.text)

time.sleep(3)

driver.quit()

실제 사이트는 https://megaitlab.com/ 입니다
본 페이지는 로그인된 페이지입니다!
뉴스: 머신러닝 강좌가 나왔어요~~~
실제 사이트는 https://megaitlab.com/ 입니다
뉴스: 머신러닝 강좌가 나왔어요~~~


#### 다음 웹 사이트에 접속해서 selenium을 활용해서 크롤링 해오세요
- https://divjason.github.io/sellenium-test/
- input tag요소에 id 값으로는 error@error.com / password 값으로는 "비밀번호1234"를 입력 후 로그인 버튼 자동클릭하기
- 클릭 이후 이동한 페이지에서 message라는 클래스 속성값을 가지고 있는 요소의 텍스트 정보를 출력!
- 해당페이지에서 로그아웃 버튼을 클릭 후 이동한 페이지에서 다시 메세지라는 클래스 속성값을 가지고 있는 요소의 텍스트 출력
- 

In [112]:
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
options.add_argument("--user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://divjason.github.io/sellenium-test/") 

element = driver.find_element(By.ID,"username")
element.clear()
element.send_keys("error@error.com")

element = driver.find_element(By.ID,"password")
element.clear()
element.send_keys("비밀번호1234")

element = driver.find_element(By.CSS_SELECTOR,"input[type='submit']") 
element.click() 
element1 = driver.find_element(By.CSS_SELECTOR,"div.message") #"태그.속성값"
print(element1.text)
time.sleep(3)
element = driver.find_element(By.CSS_SELECTOR,"input[value='로그아웃']")
element.click() 
element2 = driver.find_element(By.CSS_SELECTOR,"div.message")
print(element2.text)

time.sleep(3)

driver.quit()

실제 사이트는 https://megaitlab.com/ 입니다
본 페이지는 로그인된 페이지입니다!
뉴스: 머신러닝 강좌가 나왔어요~~~
실제 사이트는 https://megaitlab.com/ 입니다
뉴스: 머신러닝 강좌가 나왔어요~~~


In [ ]:
#선생님 해설! 로그인 + 정보값찾아오기 + 로그아웃 + 정보값찾아오기!

from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By 
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager 
from selenium.webdriver.chrome.options import Options
import time 
import pyautogui 

service = Service("C:\\Users\\pmj96\\Downloads\\chromedriver\\chromedriver.exe") 
options = Options()
options.add_argument("--disable-gpu")
options.add_argument("--start-maximized")
options.add_argument("window-size=1920x1080")
options.add_argument("lang=ko_KR")
options.add_argument("--user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36")
driver = webdriver.Chrome(service = service, options = options) 
driver.get("https://divjason.github.io/sellenium-test/") 

element = driver.find_element(By.ID,"username")
element.clear()
element.send_keys("error@error.com")

element = driver.find_element(By.ID,"password")
element.clear()
element.send_keys("비밀번호1234")

element = driver.find_element(By.CSS_SELECTOR,"input[type='submit']") 
element.click() 
element1 = driver.find_element(By.CSS_SELECTOR,"div.message") #"태그.속성값"
print(element1.text)

time.sleep(3)

driver.quit()